RecoLocalMuon/RPCRecHit/src/RPCRecHitStandardAlgo.cc

```diff
@@ -24,18 +25,32 @@ bool RPCRecHitStandardAlgo::compute(const RPCRoll& roll,
   const float fstrip = (roll.centreOfStrip(cluster.firstStrip())).x();
   const float lstrip = (roll.centreOfStrip(cluster.lastStrip())).x();
   const float centreOfCluster = (fstrip + lstrip)/2;
+  const double y = cluster.hasY() ? cluster.y() : 0;
+  Point = LocalPoint(centreOfCluster, y, 0);
```

Comment:
@slava77 of course both are accounted, maybe code is not crystal clear.
the x position correction is needed only for the Endcap chambers with y coordinate. Actually setting Point(centreOfCluster, 0, 0) in L28-29 and assigning new value something like Point(x, y, 0) in L35 is also OK since all RPC chambers with 2D information belong to the endcap.